src/views/collectable/v3/SecondaryCollectable.vue

```diff
@@ -434,6 +457,7 @@ export default {
       console.log({data})
       state.loading = false;
       state.collectable = data.collectable;
+      state.primaryCollectable = data.collectable;
```

Comment:
on line 463 in this file, the primary is merged with the secondary collectable data, which makes the slug to be overwritten